gradle/java-module.gradle

```diff
@@ -107,35 +107,19 @@ dependencies {
 		testRuntime( libraries.hana )
 	}
 
-
 	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 	// Java 9 ftw!
 	if ( JavaVersion.current().isJava9Compatible() ) {
-		// The JDK used to run Gradle is Java 9+, and we assume that that is the same
-		//		JDK for executing tasks
-		compile( libraries.jaxb_api )
-		compile( libraries.jaxb_runtime )
-		compile( libraries.jaxb_xjc )
-		compile( libraries.jaxb2_basics )
-		compile( libraries.jaxb2_basics_ant )
+		// In order for generated by jpamodelgen to be compiled successfully,
```

Comment:
I think a word is missing between  `for` and `generated`? Maybe `code`?

Reply:
Fixed